<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EnteteDetailWidget</class>
 <widget class="QWidget" name="EnteteDetailWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>816</width>
    <height>644</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EnteteDetail</string>
  </property>
  <widget class="QWidget" name="widgetCentral" native="true">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>761</width>
     <height>626</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_16">
    <item>
     <widget class="QWidget" name="widgetContainerButton" native="true">
      <property name="minimumSize">
       <size>
        <width>93</width>
        <height>621</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="pushButtonNouveau">
         <property name="text">
          <string>Nouveau</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonValider">
         <property name="text">
          <string>Valider</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonRechercher">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Rechercher</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonModifier">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Modifier</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonSupprimer">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Supprimer</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonPremier">
         <property name="text">
          <string>Premier</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonDernier">
         <property name="text">
          <string>Dernier</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonAvant">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Avant</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonArriere">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Arrierre</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonImprimer">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Imprimier</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonImprimerTout">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Imprimer tout</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonVisualiser">
         <property name="text">
          <string>Visualiser</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>314</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QScrollArea" name="scrollAreaContainerForm">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>642</width>
         <height>606</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QGroupBox" name="groupBoxEntete">
          <property name="title">
           <string>EntÃªte</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="labelCodeCmd">
               <property name="mouseTracking">
                <bool>true</bool>
               </property>
               <property name="text">
                <string>code commande</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_int_CodeCmd">
               <property name="enabled">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="labelCodeFournisseur">
               <property name="mouseTracking">
                <bool>true</bool>
               </property>
               <property name="text">
                <string>code fournisseur</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBoxCodeFournisseur">
               <property name="minimumSize">
                <size>
                 <width>150</width>
                 <height>0</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="labelDateCmd">
               <property name="mouseTracking">
                <bool>true</bool>
               </property>
               <property name="text">
                <string>date commande</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDateEdit" name="dateEditDateCmd"/>
             </item>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBoxDetail">
          <property name="title">
           <string>Detail</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="4" column="3">
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="2" column="5" colspan="6">
            <layout class="QHBoxLayout" name="horizontalLayout_15">
             <item>
              <widget class="QLabel" name="labelPrixGlobal">
               <property name="mouseTracking">
                <bool>true</bool>
               </property>
               <property name="text">
                <string>prix global</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_float_PrixGlobal">
               <property name="enabled">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_15">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item row="3" column="0" colspan="11">
            <widget class="QTableWidget" name="tableWidgetDetailDisplay">
             <property name="rowCount">
              <number>0</number>
             </property>
             <attribute name="verticalHeaderVisible">
              <bool>false</bool>
             </attribute>
             <attribute name="verticalHeaderHighlightSections">
              <bool>true</bool>
             </attribute>
             <column>
              <property name="text">
               <string>Id commande</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Ref Produit</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Prix Unitaire</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>QuantitÃ©</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Prix Global</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Date Livraison</string>
              </property>
             </column>
            </widget>
           </item>
           <item row="4" column="10">
            <widget class="QPushButton" name="pushButtonSupprimerDetail">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Supprimer</string>
             </property>
            </widget>
           </item>
           <item row="1" column="5" colspan="6">
            <layout class="QHBoxLayout" name="horizontalLayout_13">
             <item>
              <widget class="QLabel" name="labelPrixUnitaire">
               <property name="mouseTracking">
                <bool>true</bool>
               </property>
               <property name="text">
                <string>prix unitaire</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_float_PrixUnitaire"/>
             </item>
             <item>
              <spacer name="horizontalSpacer_13">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item row="4" column="2">
            <widget class="QPushButton" name="pushButtonInsererDetail">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Inserer</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QPushButton" name="pushButtonAjouterDetail">
             <property name="text">
              <string>Ajouter</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="5">
            <layout class="QHBoxLayout" name="horizontalLayout_12">
             <item>
              <widget class="QLabel" name="labelQuantite">
               <property name="mouseTracking">
                <bool>true</bool>
               </property>
               <property name="text">
                <string>quantite</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_float_Quantite"/>
             </item>
             <item>
              <spacer name="horizontalSpacer_12">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item row="0" column="0" colspan="6">
            <layout class="QHBoxLayout" name="horizontalLayout_10">
             <item>
              <widget class="QLabel" name="labelIdCmd">
               <property name="mouseTracking">
                <bool>true</bool>
               </property>
               <property name="text">
                <string>id commande</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_int_IdCmd">
               <property name="enabled">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_10">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item row="4" column="4" colspan="3">
            <widget class="QPushButton" name="pushButtonModifierDetail">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Modfier</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <spacer name="horizontalSpacer_16">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="6" colspan="5">
            <layout class="QHBoxLayout" name="horizontalLayout_11">
             <item>
              <widget class="QLabel" name="labelRefProduit">
               <property name="mouseTracking">
                <bool>true</bool>
               </property>
               <property name="text">
                <string>ref produit</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBoxRefProduit">
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>0</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_11">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item row="2" column="0" colspan="5">
            <layout class="QHBoxLayout" name="horizontalLayout_14">
             <item>
              <widget class="QLabel" name="labelDateLivraison">
               <property name="mouseTracking">
                <bool>true</bool>
               </property>
               <property name="text">
                <string>date livraison</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDateEdit" name="dateEditDateLivraison"/>
             </item>
             <item>
              <spacer name="horizontalSpacer_14">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item row="4" column="9">
            <spacer name="horizontalSpacer_5">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="4" column="8">
            <widget class="QPushButton" name="pushButtonEffacerDetail">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Effacer</string>
             </property>
            </widget>
           </item>
           <item row="4" column="7">
            <spacer name="horizontalSpacer_6">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
